zebra-rpc/src/methods.rs

```diff
@@ -94,6 +94,20 @@ pub trait Rpc {
     /// Note 3: The `verbosity` parameter is ignored but required in the call.
     #[rpc(name = "getblock")]
     fn get_block(&self, height: String, verbosity: u8) -> BoxFuture<Result<GetBlock>>;
+
+    /// getbestblockhash
+    ///
+    /// Returns the hash of the current best blockchain tip block.
+    ///
+    /// zcashd reference: <https://zcash.github.io/rpc/getbestblockhash.html>
+    ///
+    /// Result:
+    /// {
+    ///      "hex": String, // Hex encoded block hash
+    /// }
```

Comment:
Docs: the field is just a string, it doesn't have a name

```suggestion
    /// Result: [`GetBestBlockHash`]
```

Reply:
Thanks, i also fixed the `getblock` return doc in https://github.com/ZcashFoundation/zebra/pull/3754/commits/1f5768d236a141be6e634fe2878da97ada88eccc